wolox-training/mr-rails
# Kaminari

# Summary
* Added Kaminari for pagination and added ajax paginator

# Test plan
1. Go to /articles
2. Use the paginator to navigate through the articles
3. The page should not be reloaded

# Trello card
https://trello.com/c/ex0tYpzK/24-crear-lista-de-articulos-usando-api-rest
